test/management/management-token-provider.tests.js

```diff
@@ -431,7 +466,7 @@ describe('ManagementTokenProvider', function() {
   });
 
   it('should pass the correct payload in the body of the oauth/token request with cache disabled', function(done) {
-    var config = Object.assign({}, defaultConfig);
+    var config = Object.assign({}, defaultConfizg);
```

Comment:
typo? Or is this intentional for some reason?

Reply:
🤣